cmd/apps/metricsserver_app.go

```diff
@@ -134,11 +135,11 @@ func MakeInstallMetricsServer() *cobra.Command {
 
 const MetricsInfoMsg = `# Check pod usage
 
-kubectl top pod
+Kubectl top pod
 
 # Check node usage
 
-kubectl top node
+Kubectl top node
```

Comment:
I think you may have done a global search and replace, which is always dangerous. Please can you revert all of these changes and be more careful with the refactoring change?

Reply:
Yes, I mentioned it already. I had fixed it, if the general idea where good.